src/main/java/com/flippingutilities/model/OfferEvent.java

```diff
@@ -263,6 +281,7 @@ public void setTicksSinceFirstOffer(OfferEvent lastOfferForSlot)
 
 	public static OfferEvent dummyOffer(boolean buyState, boolean marginCheck, int price, int id, String itemName) {
 		return new OfferEvent(
+				UUID.randomUUID().toString(),
```

Comment:
Can't we get this to be generated by the class itself?
I know it might be a pain due to the `AllArgsConstructor` or whatever this is called, but generating the ID should be left to the class itself